src/docfx/lib/JsonUtility.cs

```diff
@@ -395,18 +410,17 @@ private static void LogInfoForNullValue(IJsonLineInfo token, List<Error> errors,
 
         private sealed class JsonContractResolver : DefaultContractResolver
         {
+            private readonly List<Error> _errors;
+
+            public JsonContractResolver(List<Error> errors = null)
```

Comment:
>null [](start = 61, length = 4)

I see this is null in `DefaultDeserializer`. Will it cause ArgumentNullException when call `converter.Validate()`?

Reply:
And also we might want to invoke our ToObject<T> instead of the one from Json.Net to execute schema validation, like config and toc

---
In reply to: [209227254](https://github.com/dotnet/docfx/pull/3175#discussion_r209227254) [](ancestors = 209227254,209203462)